The `make migrate` command does not work when installing for "dev"

When I change the .env file to `APP_ENV=prod` and install composer dependencies without the `--no-dev` flag, it seems that those additional dependencies are creating an error when installing UDOIT. 

```
make migrate
docker compose -f docker-compose.nginx.yml run php php bin/console doctrine:migrations:migrate
Symfony\Component\ErrorHandler\Error\ClassNotFoundError^ {#66
  #message: """
    Attempted to load class "WebProfilerBundle" from namespace "Symfony\Bundle\WebProfilerBundle".\n
    Did you forget a "use" statement for another namespace?
    """
  #code: 0
  #file: "./src/Kernel.php"
  #line: 23
  trace: {
    ./src/Kernel.php:23 {
      App\Kernel->registerBundles(): iterable^
      › if ($envs[$this->environment] ?? $envs['all'] ?? false) {
      ›     yield new $class();
      › }
    }
    ./vendor/symfony/http-kernel/Kernel.php:382 { …}
    ./vendor/symfony/http-kernel/Kernel.php:766 { …}
    ./vendor/symfony/http-kernel/Kernel.php:128 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:166 { …}
    ./vendor/symfony/framework-bundle/Console/Application.php:72 { …}
    ./vendor/symfony/console/Application.php:171 { …}
    ./bin/console:42 { …}
  }
}
2024-11-19T15:20:38+00:00 [critical] Uncaught Error: Class "Symfony\Bundle\WebProfilerBundle\WebProfilerBundle" not found
make: *** [migrate] Error 255
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -56,12 +56,7 @@ We provide a fast and simple way of setting up a local UDOIT instance through Do
 
 #### 2. (Optional) Install the Necessary PHP Dependencies for Dev Environment
 
-UDOIT uses Composer to install PHP dependencies. If your `APP_ENV` is set to `prod`, UDOIT will handle this for you. However, if it is set to `dev`, you may need some additional dependencies.
-
-You can set them up by running the following command:
-```
-docker compose -f docker-compose.nginx.yml run composer composer install
-```
+UDOIT uses Composer to install PHP dependencies. Running the following command will handle installing the dependencies for you.
 
 #### 3. Build the Containers
 

--- a/docker-compose.nginx.yml
+++ b/docker-compose.nginx.yml
@@ -52,7 +52,16 @@ services:
       build:
         context: ./build/nginx
         dockerfile: Dockerfile.composer
-      command: "composer install --no-dev --no-interaction --no-progress --optimize-autoloader"
+      command: >
+        sh -c '
+          if [ "$APP_ENV" = "dev" ]; then
+            echo "composer install (dev)" &&
+            composer install --no-interaction --no-progress --optimize-autoloader;
+          else
+            echo "composer install (prod)" &&
+            composer install --no-dev --no-interaction --no-progress --optimize-autoloader;
+          fi
+        '
       volumes:
         - ./:/app
       env_file:

